source/slang/slang-ir-use-uninitialized-values.cpp

```diff
@@ -265,7 +265,7 @@ namespace Slang
 
         // Consider it as a store if its passed
         // as an out/inout/ref parameter
-        auto type = as<IRType>(unwrapAttributedType(ftype->getParamType(index)));
+        auto type = cast<IRType>(unwrapAttributedType(ftype->getParamType(index)));
```

Comment:
`cast` will throw if the cast failed. should use `as` here.

Reply:
can just remove the whole thing actually :)